app/helpers/reactjs_helper.rb

```diff
@@ -5,6 +5,25 @@ def mount_react_component(name, selector, data = [], opts = {})
     end
   end
 
+  # Mount react component in views
+  # Params:
+  # +name+:: the component name from the componentRegistry
+  # +data+:: data to pass to the component as props
+  #          available values: Hash, json-string, nil
+  # +opts[:flatten_data]+:: false = will pass the props to the component props.data
+  #                         true = will pass the props directly to the component props
+  def react_component(name, data = [], opts = {})
```

Comment:
why default data to empty array if that is not a valid value type? or if it is, the comment is out of sync with the code